apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-launch.js

```diff
@@ -63,8 +67,15 @@ function WelcomeTour() {
 	// Preload card images
 	usePrefetchTourAssets( tourSteps );
 
+	let initialStepIndex = 0;
+	if ( 'sell' === intent && sitePlan && 'ecommerce-bundle' !== sitePlan.product_slug ) {
+		// There is not a good way to identify the steps
+		initialStepIndex = tourSteps.length - 2;
```

Comment:
If someone adds a step this will break.
Maybe we should add a new an extra field to the TourStep meta object so we can filter?

Reply:
Yes, I'm thinking on a `slug` field. What do you think?

Reply:
mmm... that could work... 
maybe it's clearer to add a `intent` field so if you read the tour object you get a glimpse of its usage? but just a suggestion, either one is better that offsets 😄

Reply:
now that I think about, `intent` is too broad. `slug` is better because it will allow you to point to a specific step 👍